acc-testcases/resources/network.yaml

```diff
@@ -3,12 +3,22 @@ vars:
 acc:
 - config: |
     name = "$(network_name)"
-    group_id = "shared"
-    active = true
-    dhcp_server = true
-    description = "Network created using tf"
-    cidr = "168.72.10.0/18"
-    gateway = "168.72.10.10"
-    pool_id = 6
+    description  = "Network created using tf"
+    group_id     = "shared"
+    scope_id     = "88cd4dc8-0445-4b8e-b260-0f4cd361f4e1"
+    cidr         = "172.16.10.1/24"
+    gateway      = "172.16.10.1"
+    pool_id      = 7
+    scan_network = false
+    active       = true
+    config {
+       connected_gateway = "/infra/tier-1s/26cdb82e-0057-4461-ad4d-cddd61d77b1f"
+    }
+    resource_permissions {
+      all = true
+    }
+  validations:
+    tf.cidr = "172.16.10.1/24"
+    tf.gateway = "172.16.10.1"
 
 ignore: true
```

Comment:
remove ignore, or else this test case will be skipped

Reply:
Correctly the Acceptance test willl fail with this. I tihnk we need to merge "https://github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pull/40"  to remove `ignore: true`